contracts/common/Node.sol.bak

```diff
@@ -0,0 +1,233 @@
+pragma solidity >=0.5.0 <0.6.0;
+
+import "./Input.sol";
+import "./Nibble.sol";
+import "./Bytes.sol";
+import "./Hash.sol";
+import "./Scale.sol";
+
+library Node {
```

Comment:
Will this library still be used?

`.bak`?

Reply:
Already cleaned up